### PR TITLE
Clear the error message after global.json is fixed

### PR DESCRIPTION
**Pull request by troydai**
Addressing #2060 

/cc @davidfowl @PradeepKadubandi

### CODE REVIEW

File: src/Microsoft.Dnx.DesignTimeHost/ApplicationContext.cs [hunk 1]
```diff
@@ -52,6 +52,9 @@ public class ApplicationContext
         private readonly CompilationEngineFactory _compilationEngineFactory;
         private int? _contextProtocolVersion;
 
+        private readonly Message _emptyCleanMessage;
```
Comment (davidfowl) [inline src/Microsoft.Dnx.DesignTimeHost/ApplicationContext.cs]: _noErrorMessage

Comment (troydai) [inline src/Microsoft.Dnx.DesignTimeHost/ApplicationContext.cs]: will change

**Comment by davidfowl** [comment]
So this change isn't what I would expect. I'd expect us to keep track of the last error in the World and update the remote and local world and compare them appropriately like the other things.

**Comment by troydai** [comment]
The only place we send out "Error" message is in the "catch all" in the processing loop. It is not associated or computed during update the "World".

**Comment by troydai** [comment]
Ok to sign off? Or have to associate the error with World?

**Comment by troydai** [comment]
Updated. Error is registered in remove `World`.

**Comment by troydai** [comment]
:watch:

**Comment by davidfowl** [comment]
:shipit:

**Pull request merged**